go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/bucketeer-io/bucketeer
 
-go 1.20
+go 1.21
+
+toolchain go1.21.10
```

Comment:
go-jose/v4 uses `max` function that was introduced in 1.21.0.
https://tip.golang.org/doc/go1.21